src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/EServicesApiServiceImpl.scala

```diff
@@ -423,4 +424,18 @@ final case class EServicesApiServiceImpl(
       }
     }
   }
+
+  override def getEServiceDocumentById(eServiceId: String, descriptorId: String, documentId: String)(implicit
+    contexts: Seq[(String, String)],
+    toEntityMarshallerProblem: ToEntityMarshaller[Problem],
+    toEntityMarshallerFile: ToEntityMarshaller[File]
+  ): Route = {
+    val result: Future[File] =
+      catalogProcessService.getEServiceDocumentById(eServiceId, descriptorId, documentId)(contexts)
```

Comment:
here is better to retrieve the document directly in the BFF, instead of asking it to the process.
(once done this, we could remove that endpoint from the process)

Reply:
+1 on this

Reply:
+1 on this